packages/web-app-files/tests/unit/views/PrivateLink.spec.js

```diff
@@ -0,0 +1,16 @@
+describe('PrivateLink view', () => {
+  describe('when the page has loaded successfully', () => {
+    it.todo('should have the background image set')
+    it.todo('should display the page title')
+    it.todo('should display the logo')
+    it.todo('should call "getPathForFileId" method')
```

Comment:
I'm struggling with this description. I'd like to aim for a `test driven` development approach in the future. For that we'd have to write unit tests as black box tests. A test description about whether or not a certain function was called would be not allowed in such a case. Instead you'd have to describe behaviour of the component (`should resolve the provided file id to a path from the backend`).